docs/network protocol.md

```diff
@@ -301,6 +319,56 @@ the server will forward the message to all those targets to which any one requir
 | tags | list\[str\] | Optional. Client tags that should receive this message |
 | data | dict | Any data you want to send |
 
+### Get
+Used to request a or multiple values from the server its datastorage, see the [Set](#Set) package for how to write values to the datastorage. A Get package will be answered with a [Retrieved](#Retrieved) package.
+#### Arguments
+| Name | Type | Notes |
+| ------ | ----- | ------ |
+| keys | list\[str\] | keys to retrieve the values for |
+
+### Set
+Used to write data to the server its datastorage, that data can then be shared across worlds or just saved for later. values for keys in the datastorage can be retrieved with an [Get](#Get) package, or monitored with an [SetNotify](#SetNotify) package.
+#### Arguments
+| Name | Type | Notes |
+| ------ | ----- | ------ |
+| key | str | The key to manipulate  |
+| value | any | The new value for the key  |
+| default | any | The default value to use incase the key has no value on the server |
+| want_reply | bool | if set, will send an [SetReply](#SetReply) response back to the client |
+| operations | list\[[DataStorageOperation](#DataStorageOperation)\] | operations to apply to the value, multiple operations can be present they will be executed in order of appearance |
+
+Additional arguments send in this package will also be added to the [SetReply](#SetReply) packege it triggers
+
+#### DataStorageOperation
+An datastorage operation manipulates or alters the value of a key in the datastorage. If the operation transform's the value from one state to an other then the current value of the key is used as starting point or the [Set](#Set)'s package `default` if the key does not exist on the server.
```

Comment:
```suggestion
A DataStorageOperation manipulates or alters the value of a key in the data storage. If the operation transforms the value from one state to another then the current value of the key is used as the starting point otherwise the [Set](#Set)'s package `default` is used if the key does not exist on the server already.
```